eth/handler.go

```diff
@@ -674,3 +675,11 @@ func (h *handler) txBroadcastLoop() {
 		}
 	}
 }
+
+// setSynced sets the flag as true when node finishes chain sync.
+func (h *handler) setSynced() {
```

Comment:
setSynced is an unfortunate name IMO because it doesn't really set any sync status, rather it reconfigures the node into a different mode of operation post-sync. Perhaps enableSyncedFeatures()? Seems like a mouthful, but at least somewhat consistent with what it does?